tests/browser_test.go

```diff
@@ -50,6 +52,23 @@ func TestBrowserNewPage(t *testing.T) {
 	assert.Equal(t, 0, l, "expected there to be 0 browser context after second page close, but found %d", l)
 }
 
+func TestTmpDirCleanup(t *testing.T) {
+	tmpDirPath := "./"
+
+	err := os.Setenv("TMPDIR", tmpDirPath)
+	assert.NoError(t, err)
+	defer func() {
+		err = os.Unsetenv("TMPDIR")
+		assert.NoError(t, err)
+	}()
+
+	b := newTestBrowser(t)
+	_ = b.NewPage(nil)
+
+	_, err = os.Stat(tmpDirPath + "xk6-browser-data-*")
+	assert.True(t, errors.Is(err, os.ErrNotExist), tmpDirPath+"xk6-browser-data-* should have been deleted")
```

Comment:
This test does not seem to check whether the temporary directory is removed.

I just returned `nil` from the `DataStore.Cleanup` method, and it still worked 😊

Reply:
Just realised that `os.Stat` doesn't work with patterns so now working with `filepath.Glob` which does

Reply:
Great! Please let me know by re-requesting my review when you finish the whole PR so I can look at the whole thing in one Go.